airflow/www/templates/airflow/dags.html

```diff
@@ -159,7 +159,7 @@ <h2>{{ page_title }}</h2>
         <form id="tags_form">
           <div class="form-group search-input">
             <select multiple name="tags" id="tags_filter" class="select2-drop-mask">
-              {% for tag in tags %}
+              {% for tag in tags.sort() %}
```

Comment:
This will sort but there is no key specified, will this sort on the DagTag.id ? Is it what we want, maybe the name is more relevant ?

Reply:
Ah you are absolutely right, it should indeed be sorted by name. Looking into that now.

Reply:
@pierrejeambrun Would it not be worthwhile to change this in views.py? 
Say addition of:
`tags = sorted(tags, key=lambda t: t['name'])`

Reply:
Just tested this out locally and it works as expected, I think this change to views.py is all that is needed.

Reply:
![image](https://github.com/apache/airflow/assets/102953522/ee4c9b91-5edf-4336-8835-afb9222eae2f)

Reply:
Can you `order_by` directly in the query then ? There is no need to iterate an extra time just to order it.

Reply:
@pierrejeambrun Done, I've tried it out locally and it looks good there.